src/api/documents/documents.service.ts

```diff
@@ -1,12 +1,43 @@
-import { Injectable, HttpException, HttpStatus } from '@nestjs/common';
 import * as fs from 'fs';
+import * as fsPromises from 'fs/promises';
 import * as util from 'util';
+import * as path from 'path';
+import { Injectable, HttpException, HttpStatus } from '@nestjs/common';
+
+export const RULES_DIRECTORY = 'brms-rules/rules';
```

Comment:
Could we make this an environmental variable, or would that create conflict with the github workflow? 
Could make it easier to update location in the future?

Reply:
Yeah, good point. I was being a bit lazy here. I found a way to add it as an environment variable that has a default in nestjs and can be used in the unit tests. Updated!